src/app/pages/addons/MarkerWithLabelExample.js

```diff
@@ -0,0 +1,49 @@
+/* global google */
+
+import {
+  default as React,
+  Component,
+} from "react";
+
+import {
+  withGoogleMap,
+  GoogleMap,
+  Marker,
+} from "../../../lib";
+
+import styles from "./markerWithLabel.css"
+
+const MarkerWithLabelExampleMap = withGoogleMap(props => (
+  <GoogleMap
+    defaultZoom={3}
+    defaultCenter={{lat: 25.0391667, lng: 121.525}}
+  >
+    <Marker
+      markerWithLabel={window.MarkerWithLabel}
+      position={{
+        "lat": 25.0391667,
+        "lng": 121.525,
+      }}
+      opacity={0}
+      labelClass="map-price-container"
+      labelAnchor={new google.maps.Point(35, 27)}
+      labelContent={`<div class="map-price-marker "><span>135.123.000 $</span></div>`}
```

Comment:
whitespace after classname, and if we could display the dollar value as $135,123.00, that would be ideal or use a different value for the label.

eg:
`<div class="map-price-marker"><span>$ 135,123.00</span></div>`